src/test/java/bio/terra/janitor/service/pubsub/TrackedResourceSubscriberTest.java

```diff
@@ -42,7 +42,7 @@
   @Qualifier(OBJECT_MAPPER)
   private ObjectMapper objectMapper;
 
-  @Autowired private JanitorService janitorService;
+  @Autowired private TrackedResourceService trackedResourceService;
 
   @Autowired private MockMvc mvc;
```

Comment:
I don't think this test should have an authenticated user or MockMvc.

Reply:
Done